docs/dev/reference/dca/list.md

```diff
@@ -32,6 +32,7 @@ $GLOBALS['TL_DCA']['tl_example']['list']['sorting'] = [
 | rootPaste             | true/false (`bool`)           | Enable paste buttons at root level. (default: false)                                                                                                                                                                                                                                                                                                                             |
 | filter                | Query filter (`array`)           | Allows you to add custom filters as arrays, e.g. `[['status=?', 'active'], ['usages>?', 0]]`.                                                                                                                                                                                                                                                                                                                        |
 | disableGrouping       | true/false (`bool`)           | Allows you to disable the group headers in list view and parent view.                                                                                                                                                                                                                                                                                                                                  |
+| defaultSearchField    | Field name (`string`)           | Set a default search field for the search records menu instead of selecting the alphabetically first field.                                                                                                                                                                                                                                                                                                                          |
```

Comment:
```suggestion
| defaultSearchField    | Field name (`string`)           | {{< version-tag "5.1" >}} Set a default search field for the search records menu instead of selecting the alphabetically first field.                                                                                                                                                                                                                                                                                                                          |
```